script.js

```diff
@@ -0,0 +1,30 @@
+//Declare and initilize variables
+let howOldAreYou;
+let invalidInput;
+let Age;
+let approvedAge = 21;
+const WELCOME_MESSAGE = `Welcome To The Kosher Wine Cellar!
+You must be over 21 years to purchase on our site!`;
+
+alert(WELCOME_MESSAGE);
+
+Age = prompt('Please Enter Your Age');
+
+console.log(typeof Age);
+
+invalidInput = isNaN(Age);
+
+Age =Number(Age);
```

Comment:
```suggestion
age = Number(age);
```

You can also do

```
age = +age;
```

Using the plus operator next to a string will convert it to a number